nav.js

```diff
@@ -0,0 +1,28 @@
+const now = new Date();
+const dateTime = document.querySelectorAll('span');
+for (let i = 0; i < dateTime.length; i += 1) {
+  dateTime[i].innerHTML = now.toString();
+}
+dateTime.innerHTML = now.toString();
```

Comment:
- [ ] good job setting the date and time in your application 👍 however according to [the requirements ](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#:~:text=Display%20the%20current,date%20and%20time.)you should use the Luxon library to set it.